src/renderer/react/components/general/button/Button.jsx

```diff
@@ -19,7 +19,7 @@ Button.propTypes = {
   /** Boolean indicating whether the button should render as disabled */
   disabled: PropTypes.bool,
   /** Button label. */
-  label: PropTypes.string.isRequired,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```

Comment:
PropTypes.string is already included in PropTypes.node

Reply:
Resolved